packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -55,19 +89,42 @@ export type SelectOption<T> = {
   [key: string]: unknown;
 } & CommonProps;
 
+export type SelectValueChangeEvent = 'Added' | 'Removed';
+
+export type MultipleType<T> =
```

Comment:
if we are exporting this, we should use a more descriptive name. `MultipleType` sounds too vague

or don't export it

Reply:
I exported only for tests :/
Changed the naming a bit, not sure if its better now